physics/GFS_rrtmg_pre.F90

```diff
@@ -608,10 +608,10 @@ subroutine GFS_rrtmg_pre_run (im, levs, lm, lmk, lmp, n_var_lndp,        &
        if (aero_dir_fdb) then
          do k=1,lmk
            do i=1,im
-             aer_nm(i,k,1 )=aer_nm(i,k,1 )+qgrs(i,k,ntdust )*0.11
-             aer_nm(i,k,2 )=aer_nm(i,k,1 )+qgrs(i,k,ntdust )*0.89
-             aer_nm(i,k,12)=aer_nm(i,k,12)+qgrs(i,k,ntsmoke)*0.01
-             aer_nm(i,k,14)=aer_nm(i,k,14)+qgrs(i,k,ntsmoke)*0.99
+             aer_nm(i,k,1 )=aer_nm(i,k,1 )+qgrs(i,k,ntdust )*1.e-9*0.11
```

Comment:
Please use consistent way as GFS to add the tracers from chemical model for radiative forcing: https://github.com/NCAR/ccpp-physics/pull/910/commits/a02422c45b06f46cf0a61ccfcd3083666b2918d0 , by this using way, the GFS will not be impact by the order of chemical tracers, which would be safe for long-term use.

Reply:
@zhanglikate Thanks for your suggestions. Yes, our way is similar to that in https://github.com/NCAR/ccpp-physics/commit/a02422c45b06f46cf0a61ccfcd3083666b2918d0. We treat part of smoke/dust as anomaly, and add on top of the climatology. We will further tune the magnitude in our following test runs.